packages/nextjs/app/blockexplorer/address/[address]/page.tsx

```diff
@@ -76,8 +76,14 @@ const getContractData = async (address: string) => {
   return { bytecode, assembly };
 };
 
+export function generateStaticParams() {
+  return [{ address: "0x0000000000000000000000000000000000000000" }];
+}
+
 const AddressPage = async ({ params }: PageProps) => {
   const address = params?.address as string;
+  if (address === "0x0000000000000000000000000000000000000000") return null;
```

Comment:
and `if (!address)` here. And the same for transactions

Tested and everything works fine.

Not very good too but looks a bit better. Will continue to try to find something better today, but not sure it's possible

Reply:
> and if (!address) here. And the same for transactions

If are not able to do https://github.com/scaffold-eth/scaffold-eth-2/pull/825#discussion_r1576845520, then this suggested if check won't work right ? since we need to have some kind of valid string passed from `generateStaticParams`

Maybe I can extract that zero address in util, since sometimes it handy for other stuff too

Reply:
> Maybe I can extract that zero address in util, since sometimes it handy for other stuff too

I like this too!

It sounds like something viem could implement